docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,739 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        cands_NN = np.empty(n_cands, dtype='int64')
</pre><p>Please use <code>np.int64</code>  to be consistent with STUMPY dtype declaration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Will be fixed in next commits.